Mathlib/GroupTheory/GroupAction/Transitive.lean

```diff
@@ -0,0 +1,93 @@
+/-
+Copyright (c) 2022 Antoine Chambert-Loir. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Antoine Chambert-Loir
+-/
+import Mathlib.GroupTheory.GroupAction.Defs
+import Mathlib.GroupTheory.GroupAction.Hom
+
+/-! # Complements to pretransitive actions
+
+When `f : X →ₑ[φ] Y` is an equivariant map with respect to a map
+of monoids `φ: M → N`,
+
+- `IsPretransitive.of_surjective_map` shows that
+  the action of `N` on `Y` is pretransitive
+  if that of `M` on `X`  is pretransitive.
+
+- `IsPretransitive.of_bijective_map` shows that when
+  `φ` is surjective, the action of `N` on `Y` is pretransitive
+  iff that of `M` on `X`  is pretransitive.
+
+Given `MulAction G X` where `G` is a group,
+
+- `IsPretransitive.mk_base G a` shows that `IsPretransitive G X`
+  iff every element is translated from `a`
+
+- `IsPretransitive.iff_orbit_eq_top G a` shows that `IsPretransitive G X`
+  iff the `orbit G a` is full.
+
+-/
+
+variable {G : Type*} [Group G] {X : Type*} [MulAction G X]
+
+namespace MulAction.IsPretransitive
+
+open MulAction
+
+/-- An action of a group is pretransitive iff any element can be moved from a fixed given one. -/
+@[to_additive
+  "An additive action of an additive group is pretransitive
+  iff any element can be moved from a fixed given one."]
+theorem mk_base_iff (a : X) :
+    IsPretransitive G X ↔ ∀ x : X, ∃ g : G, g • a = x := by
```

Comment:
I see where this name is coming from but I don't think it is discoverable. What about `MulAction.isPretransitive_iff_base`?